zap/src/main/java/org/zaproxy/zap/scan/filters/FilterCriteria.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2019 The ZAP Development Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.zap.scan.filters;
+
+/**
+ * @author KSASAN preetkaran20@gmail.com
+ *     <p>Processing of FilterCriteria is first Include then Exclude and then Include_All so in case
```

Comment:
The documentation should be before the author tag (we don't maintain/use author tags, it could be removed).

Reply:
sure will move documentation above author tag.